src/Amazon.Extensions.CognitoAuthentication/Amazon.Extensions.CognitoAuthentication.csproj

```diff
@@ -46,8 +46,8 @@
   </Choose>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.CognitoIdentity" Version="[3.3.100, 4.0)" />
-    <PackageReference Include="AWSSDK.CognitoIdentityProvider" Version="[3.3.100, 4.0)" />
+    <PackageReference Include="AWSSDK.CognitoIdentity" Version="3.5.0.25" />
```

Comment:
This will break for the customer who are using `AWSSDK.CognitoIdentity` older than `3.5.0.25`.

Reply:
Yes, since customers will have to use the latest version, we are also doing a major version bump